Write up documentation on posthog-dotnet (#10589)
PostHog/posthog.com: contents/docs/integrate/_snippets/install-dotnet.mdx

<DotNetInstall />

At the moment, ASP.NET Core is the only supported platform for the PostHog .NET SDK.

```bash
dotnet add package PostHog.AspNetCore
```

In your `Program.cs` file, add the following code:

```csharp
using PostHog.AspNetCore;

var builder = WebApplication.CreateBuilder(args);

builder.AddPostHog();
```

> Make sure to configure PostHog with your project API key, instance address, and optional personal API key. For example, in appsettings.json:

Reviewer: ```suggestion
Make sure to configure PostHog with your project API key, API host, and optional personal API key (which you can get from [your project settings](https://us.posthog.com/settings/project)). For example, in `appsettings.json`:
```

```json
{
  "PostHog": {

>     "ProjectApiKey": "phc_...",
>     "Host": "https://us.i.posthog.com"

Reviewer: These will automatically fill with values if they exist
```suggestion
    "ProjectApiKey": "<ph_project_api_key>",
    "Host": "<ph_client_api_host>"
```

  }
}
```

> **Note:** If the host is not specified, the default host `https://us.i.posthog.com` is used.

> Use a secrets manager to store your personal API key. For example, when developing locally you can use the `UserSecrets` feature of the `dotnet` CLI:

Reviewer: ```suggestion
Adding a personal API key is optional, but enables you to [locally evaluate feature flags](/docs/feature-flags/local-evaluation). Use a secrets manager to store your personal API key. For example, when developing locally you can use the `UserSecrets` feature of the `dotnet` CLI:
```

```bash
dotnet user-secrets init

> dotnet user-secrets set "PostHog:PersonalApiKey" "phc_..."

Reviewer: ```suggestion
dotnet user-secrets set "PostHog:PersonalApiKey" "phx_..."
```

```

> You can find your project API key and instance address in the [project settings](https://app.posthog.com/project/settings) page in PostHog.

Reviewer: ```suggestion
```

> To see detailed logging, set the log level to `Debug` or `Trace` in appsettings.json:

Reviewer: ```suggestion
To see detailed logging, set the log level to `Debug` or `Trace` in `appsettings.json`:
```
